backend/scripts/seed_db.sh

```
@@ -61,22 +43,33 @@ parse_arguments() {
-  python -m aci.cli.aci create-random-api-key --visibility-access public --org-id 107e06da-e857-4864-bc1d-4adcba02ab76
-fi
+seed_test_apps() {
+  # Create a temporary file
+  temp_oauth2_secrets_file=$(mktemp)
 
-# Seed the database with Plans
-if [ "$SEED_PLANS" = true ]; then
-  python -m aci.cli populate-subscription-plans --skip-dry-run
-fi
+  # Make sure it gets deleted when the script exits
+  trap "rm -f $temp_oauth2_secrets_file" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)